lib/new_relic/agent/attribute_filter.rb

```diff
@@ -124,8 +124,6 @@ def prep_attributes_exclude_rules(config)
 
       def prep_capture_params_rules(config)
         build_rule(['request.parameters.*'], include_destinations_for_capture_params(config[:capture_params]), true)
-        build_rule(['job.resque.args.*'], include_destinations_for_capture_params(config[:'resque.capture_params']), true)
```

Comment:
These attributes are no longer treated special and separate, they now work exactly like any other attribute given to the `attribute.include` config.